apps/avatax/integration/tests/product_without_taxes.spec.ts

```diff
@@ -0,0 +1,176 @@
+import { it } from "vitest";
+import { handler, spec } from "pactum";
+import { string } from "pactum-matchers";
+
+// Trick editor into highlighting gql, wihtout parsing
+const gql = String.raw;
+
+it("should return taxes for product without taxes", async () => {
+  await spec()
+    .post("/graphql/")
+    .withGraphQLQuery(gql`
+      mutation CreateCheckout(
```

Comment:
Can you check if it is possible to keep the requests in a separate file and call only the name here, e.g. CreateCheckoutWithFullInfo, createCheckoutBillingAddressOnly, etc.?

Reply:
yup, I'm not sure how we'll do this - once I add more requests then probably adding them into a separate directory will be a good idea 😅